adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMediationAdapterVungle.m

```diff
@@ -42,11 +42,13 @@ @implementation GADMediationAdapterVungle {
 
 + (void)setUpWithConfiguration:(nonnull GADMediationServerConfiguration *)configuration
              completionHandler:(nonnull GADMediationAdapterSetUpCompletionBlock)completionHandler {
+  // Gather all app ids supplied by the server configuration
   NSMutableSet *applicationIDs = [[NSMutableSet alloc] init];
-
   for (GADMediationCredentials *cred in configuration.credentials) {
     NSString *appID = cred.settings[GADMAdapterVungleApplicationID];
-    GADMAdapterVungleMutableSetAddObject(applicationIDs, appID);
+    if (appID) {
```

Comment:
Just want to confirm if we need to do `if (appID.length)`.

Reply:
Our SDK has a check internally to see if the string is empty and send an error, but I think your appID.length would handle more integration mishaps. I'll change it.